R/phenotypes.R

```diff
@@ -277,6 +277,177 @@ setPheno = function(pop, h2=NULL, H2=NULL, varE=NULL, corE=NULL,
   return(pop)
 }
 
+#' @title Set population-level phenotypes
+#'
+#' @description
+#' Compute and store population-level phenotypic summaries by applying a
+#' user-supplied function to the individual-level phenotype matrix.
+#' For a \code{MultiPop} object, the function is applied independently to
+#' each population.
+#'
+#' @param pop An object of class \code{\link{Pop-class}},
```

Comment:
```suggestion
#' @param x An object of class \code{\link{Pop-class}},
```

Change to denote that `x` can be either `Pop`, `HybridPop`, or `MultiPop` class?

Reply:
also below in the code

Reply:
Also addressed in 2a672835f5919c8f2622be379c8792116cdfb98f